src/translators/sql_translator.py

```diff
@@ -445,7 +444,7 @@ def _insert_original_entities_in_failed_batch(
         stmt = f"insert into {table_name} ({cols}) values (?, ?, ?, ?)"
         tix = current_timex()
         batch_id = uuid4().hex
-        rows = [[entity_id(e), entity_type(e), tix,
+        rows = [[e.get('id'),  e.get('type'),  tix,
```

Comment:
shouldn't this be the other way around? i.e. keep the row you've deleted and delete the row you've added? i think that was the intent of the todo up in the file, see my other comment about it?